src/main/app/index.js

```diff
@@ -217,6 +252,33 @@ class App {
       this.createEditorWindow()
     })
 
+    ipcMain.on('screen-capture', win => {
+      if (isOsx) {
+        // Use macOs `screencapture` command line when in macOs system.
+        const { screenshotFileName } = this
+        exec(`screencapture -i -c`, async (err) => {
+          if (err) {
+            log.error(err)
+            return
+          }
+          try {
+            // Write screenshot image into screenshot folder.
+            const image = clipboard.readImage()
+            const bufferImage = image.toPNG()
+            await fse.writeFile(screenshotFileName, bufferImage)
+          } catch (err) {
+            log.error(err)
+          }
+          win.webContents.send('mt::screenshot-captured')
+        })
+      } else {
+        if (this.shortcutCapture) {
+          this.lanchScreenshotWin = win
+          this.shortcutCapture.shortcutCapture()
```

Comment:
I use the system screen shot on macOs, but I don't know wheather there is a similar command line, so I used a library, I think you can help with this if there are similar system tools in Linux and Windows?

Reply:
I think we should show the menu only on macOS then because Linux screen capture depends on the desktop environment or installed packages, Windows one is a build in app.

Reply:
OK!

Reply:
I just comment `shortcut-capture` codes, feel free to enable it after you test it on Windows or Linux if we need screen capture on Windows and Linux.